<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeteoRunwayDialog</class>
 <widget class="QDialog" name="MeteoRunwayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Meteo Runway Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="runwayName">
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="controlTime">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">M</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="magneticWind">
     <property name="inputMask">
      <string notr="true">999;-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_16">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="averageSpeed">
     <property name="inputMask">
      <string notr="true">99;-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">G</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLineEdit" name="maxSpeed">
     <property name="inputMask">
      <string notr="true">99;-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="label_17">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="QLineEdit" name="sideWind">
     <property name="inputMask">
      <string notr="true">99.9;-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="label_18">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="0" column="13" colspan="3">
    <widget class="QLineEdit" name="longitudialWind">
     <property name="inputMask">
      <string notr="true">#99;-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of runway:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Time of observation (HHMM):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Magnetic Wind Direction (degrees):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Average and maximum speed (meters per second):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="10" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Crosswind component relative to the runway (meters per second):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="13" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>The longitudinal component of the wind relative to the runway (meters per second):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="sightDistance">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_19">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="sDistanceStart">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_20">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="sDistanceMiddle">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_21">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLineEdit" name="sDistanceEnd">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7" colspan="2">
    <widget class="QLineEdit" name="lowerCloudiness">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9" colspan="2">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="2" column="11">
    <widget class="QLineEdit" name="pressureMM">
     <property name="inputMask">
      <string notr="true">999;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="12">
    <widget class="QLabel" name="label_22">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="2" column="13">
    <widget class="QLineEdit" name="pressgPa">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="14">
    <widget class="QLabel" name="label_23">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">(</string>
     </property>
    </widget>
   </item>
   <item row="2" column="15">
    <widget class="QLineEdit" name="pressureSea">
     <property name="inputMask">
      <string notr="true">9999;-</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="16">
    <widget class="QLabel" name="label_24">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Meteorological visibility on runway (meters):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Meteorological visibility JVI, beginning-, end-, mid-runway (meters):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="7" colspan="4">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>The height of the cloud base (meters):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="11" colspan="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Pressure at work the runway threshold (QFE) (mm. Hg. Art., hectopascals):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="15">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Pressure reduced to sea level (hectopascals):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="temperature">
     <property name="inputMask">
      <string notr="true">#99.9;-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_25">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">/</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="condensationPoint">
     <property name="inputMask">
      <string notr="true">#99.9;-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Air temperature and dew point (degrees):</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="9" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeteoRunwayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeteoRunwayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
